audit.json

```diff
@@ -170,5 +170,10 @@
   "10096_Timestamp Disclosure - Unix_http://em-anno-aat.service.core-compute-aat.internal/api/comments_POST":"ignore",
   "10096_Timestamp Disclosure - Unix_http://em-anno-aat.service.core-compute-aat.internal/api/metadata/_POST":"ignore",
   "10096_Timestamp Disclosure - Unix_http://em-anno-aat.service.core-compute-aat.internal/api/bookmarks_PUT":"ignore",
-  "100000_A Client Error response code was returned by the server_http://em-anno-aat.service.core-compute-aat.internal/v3/api-docs_GET": "ignore"
+  "100000_A Client Error response code was returned by the server_http://em-anno-aat.service.core-compute-aat.internal/v3/api-docs_GET": "ignore",
+  "10010_Cookie No HttpOnly Flag_http://em-anno-aat.service.core-compute-aat.internal/v2/api-docs_GET":"ignore",
+  "10054_Cookie without SameSite Attribute_http://em-anno-aat.service.core-compute-aat.internal/v2/api-docs_GET":"ignore",
+  "10096_Timestamp Disclosure - Unix_http://em-anno-aat.service.core-compute-aat.internal/v2/api-docs_GET":"ignore",
+  "100000_A Client Error response code was returned by the server_http://em-anno-aat.service.core-compute-aat.internal/v2/api-docs_GET":"ignore",
```

Comment:
you do not need these. this issues fixed. you can close this pr.

Reply:
Just checked em-annotation-api build and I noticed still security issue exists.
https://build.platform.hmcts.net/view/Evidence/job/HMCTS_Nightly_EM/job/em-annotation-api/job/master/